docs/interactive_engine/getting_started.md

```diff
@@ -6,11 +6,13 @@ This guide gives you a quick start to use GraphScope for graph interactive tasks
 
 GIE works as the core component of GraphScope. Therefore, you should install GraphScope at first to use GIE.
 
-GraphScope provides python interface which requires:
+GraphScope requires the following software versions:
 
-- Python3.7 ~ 3.11
+- Python 3.9 ~ 3.11
+- Java 11 (both Java 8 and 20 have compatibility issues)
```

Comment:
Maybe it would be better to list the environment requirement for all engines in the same place? it would be easier to keep them consistent.

e.g., you'll find that the following page is saying we only require Java 8 for GIE.

https://graphscope.io/docs/deployment/install_on_local#prerequisites

Reply:
Make sense. But shall we include an extra page for prerequisites. It 's actually required everywhere, and there are much more requirements than we've listed in https://graphscope.io/docs/deployment/install_on_local#prerequisites.

Reply:
@sighingnow I have already tried, java 8 would not work for gie any more. LoL